easypost/__init__.py

```diff
@@ -61,8 +60,7 @@
 # config
 api_key = None
 api_base = "https://api.easypost.com/v2"
-# use our default timeout, or our max timeout if that is less
-timeout = min(60, _max_timeout)
+timeout = _platform_timeout
```

Comment:
I don't see any reason for a distinction between `_platform_timeout`/`timeout` to exist now. Can you remove this line and rename _platform_timeout to timeout on lines 26 and 28?

Reply:
I initially did and put it back for fear that others would need/want the `urlfetch.get_default_fetch_deadline()` bit. In the off chance the GAE has a lower deadline, we'd want to get that instead.

Reply:
Leave it as `timeout = urlfetch.get_default_fetch_deadline() or TIMEOUT` and just don't redirect the variable name.

Reply:
It is where it is because `urlfetch` isn't used by default (requests is). We only run that line of code if we import that module, otherwise (when not in a GAE context), we use the normal timeout.

Reply:
Ah, thanks for clarifying offline. Updated the variable per your suggestion.